.github/workflows/build-and-publish.yml

```
@@ -45,7 +45,7 @@ jobs:
           mvn versions:set -DnewVersion="$PACKAGE_VERSION"
 
       - name: Release Maven package
-        run: mvn deploy --batch-mode --settings "/home/runner/.m2/settings.xml"
+        run: mvn deploy --batch-mode --settings "/home/runner/.m2/settings.xml" -Dgpg.passphrase=${{ secrets.GPG_PASSPHRASE }}
```

```suggestion
        run: mvn deploy --batch-mode --settings "/home/runner/.m2/settings.xml" -Dgpg.passphrase="${{ secrets.GPG_PASSPHRASE }}"
```

Sorry, one more change - let's add quotes around the secret (we do it for the secrets run in mvn commands in the [pact test workflow](https://github.com/alphagov/pay-ci/blob/master/.github/workflows/_run-provider-pact-tests-for-consumer.yml)).